Xpense/urls.py

```diff
@@ -14,8 +14,10 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', include('house.urls')),
+    # path('tips/', include('tips.urls')),
```

Comment:
I don't think a comment is a good approach. 
you can create a new PR which is based on this one with that change.

Reply:
I left it with a comment to explain how to add the routing for the tips app

Reply:
Ok, so you can remove that?